suse2013/common/trim-verbatim.xsl

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Purpose:
+     Trim start/end of verbatim areas (screen, programlisting, synopsis)
+
+
+    Author:     Stefan Knorr <sknorr@suse.de>
+    Copyright:  2015
+
+-->
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
```

Comment:
Should we have a parameter to switch this behaviour on and off? Just in case something goes horribly wrong...